metrics/web/test/common/presentation/routes/metrics_page_route_test.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:metrics/common/presentation/routes/metrics_page_route.dart';
+import 'package:metrics/common/presentation/routes/route_name.dart';
+import 'package:test/test.dart';
+
+import '../../../test_utils/matcher_util.dart';
+
+void main() {
+  group('MetricsPageRoute', () {
+    test(
+      'constructor throws the AssertionError if the builder is null',
```

Comment:
Can we remove the `constructor` from the description to be consistent with other tests in the project?
```suggestion
      'throws the AssertionError if the builder is null',
```